examples/integrations/yolo/import_yolo_annotations.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>### Import YOLOv8 Annotations to a Project</p><p> </p><p>Now that you have finished your initial setup, we can create predictions using YOLOv8 and import the annotations into our project. In this step, we will:</p><p>1. Define our import functions</p><p>2. Create our labels</p><p>3. Import our labels as either ground truths or MAL labels (pre-labels)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Done